<launch>        
        <!-- Motor Controller Node -->

        <!-- Arguments (Overrideable)-->
        <arg name="MotorForwardService" default="setMotorForward"/>
        <arg name="MotorReverseService" default="setMotorReverse"/>
        <arg name="StopMotorsService" default="stopAllMotors"/>
        <arg name="StopMotorService" default="stopMotor"/>
        <arg name="getRPMService" default="getRPM"/>
        <arg name="setPWMService" default="setPWM"/>
        <arg name="CalibrateMotor" default="calibrateMotor"/>
        <arg name="motor_controller_fd" default="/dev/sttyUSB0"/>

        <node name="motor_controller" pkg="motor_controller" type="motor_controller" required="true">
          <param name="MotorForwardService" value="$(arg MotorForwardService)"/>
          <param name="MotorReverseService" value="$(arg MotorReverseService)"/>
          <param name="StopMotorsService" value="$(arg StopMotorsService)"/>
          <param name="StopMotorService" value="$(arg StopMotorService)"/>
          <param name="getRPMService" value="$(arg getRPMService)"/>
          <param name="setPWMService" value="$(arg setPWMService)"/>
          <param name="CalibrateMotor" value="$(arg CalibrateMotor)"/>
          <param name="motor_controller_fd" value="$(arg motor_controller_fd)"/>
        </node>

</launch>
